doc/user_manual/postprocessor.tex

```diff
@@ -2084,3 +2085,46 @@ \subsubsection{CrossValidation}
     pseudo-random number generator state used for shuffling. If None, use default numpy RNG for shuffling.
 \end{itemize}
 
+
+%%%%%%%%%%%%%% ValueDuration %%%%%%%%%%%%%%%%%%%
+\subsubsection{ValueDuration}
+\label{ValueDurationPP}
+The \xmlNode{ValueDuration} postprocessor is a tool to construct a particular kind of histogram, where the
+independent variable is the number of times a variable exceeds a particular value, and the dependent variable
+is the values themselves.  An example of this is the Load Duration Curve in energy modeling. This approach is
+similar to that used in Lebesgue integration. Note that for each realization in the input
+\xmlNode{HistorySet}, a seperate load duration curve will be created for each target.
+
+The \xmlNode{ValueDuration} postprocessor can only act on \xmlNode{HistorySet} data objects, and generates a
+\xmlNode{HistorySet} in return.  Two output variables are created for each \xmlAttr{target}:
+\xmlString{counts\_x} and \xmlString{bins\_x}, where \xmlString{x} is replaced by the name of the target.
+These must be specified in the output data object in order to be collected.
+
+To plot a traditional Load Duration Curve, the x-axis should be the bins variable, and the y-axis should be
+the counts variable.
+
+\ppType{ValueDuration}{ValueDuration}
+%
+\begin{itemize}
+  \item \xmlNode{target}, \xmlDesc{comma separated string, required field}, specifies the names of the
```

Comment:
strings

Reply:
If we intend this to be a standard, it should be (1) documented, and (2) wrapped as a variable (like \csstring) to assure uniformity.  Changed.

Reply:
Yes, it is a documentation standard (unwritten rule we are supposed to follow since the first version of this manual) . We can add variables for this (and all the other things). 

Please let's remember to discuss about this in the Tuesday (not Monday this time) meeting.